dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -86,6 +86,9 @@ RUN apt-get -y purge   \
 {{ install_debian_packages(docker_base_stretch_debs.split(' ')) }}
 {%- endif %}
 
+# For supervisord
```

Comment:
Suggest to make more descriptive: `# Add support for supervisord to handle startup dependencies`